pyproject.toml

```diff
@@ -12,6 +12,7 @@ appdirs = "*"
 PySide6 = "^6.4.1"
 colorama = "*"
 pyxdg = {version = "*", platform = "linux"}
+python-magic = "^0.4.27"
```

Comment:
Unless we plan to run the dummy conversion for testing, I don't think we need this here.

Reply:
You're right. I used this as a temporary solution and forgot to remove it.

Reply:
Fixed via commit `543f0c9`